frontends/web/src/routes/buy/moonpay.tsx

```diff
@@ -85,7 +87,7 @@ class Moonpay extends Component<Props, State> {
 
     public render() {
         const { moonpay, t } = this.props;
-        const { height } = this.state || { height: 500 }; //Check why state is null (Load HOC?)
+        const { height } = this.state; //Check why state is null (Load HOC?)
```

Comment:
I think the comment is not needed anymore, correct?
please remove

Reply:
my bad